R/database.R

```diff
@@ -2404,13 +2509,15 @@ get_ssm_by_region = function(chromosome,
 #'
 #' @details Effectively retrieve coding SSM calls. Multiple filtering parameters are available for this function.
 #' For more information on how to implement the filtering parameters, refer to the parameter descriptions as well as examples in the vignettes.
+#' Is this function not what you are looking for? Try one of the following, similar, functions; [GAMBLR::get_coding_ssm], [GAMBLR::get_coding_ssm_status],
+#' [GAMBLR::get_ssm_by_patients], [GAMBLR::get_ssm_by_sample], [GAMBLR::get_ssm_by_samples], [GAMBLR::get_ssm_by_region], [GAMBLR::get_ssm_by_regions]
```

Comment:
Looks like in this list this description refers back to the function it describes (`get_coding_ssm`)?

Reply:
Good catch! It has been fixed.